code/datalab-app/src/api/corsConfig.js

```diff
@@ -0,0 +1,25 @@
+import config from './config';
+
```

Comment:
Is it worth bringing some of the comment from the snippet so we remember why we have the function?